test/testdrive/mz-arrangement-sharing.td

```diff
@@ -724,6 +724,20 @@ ReduceMinsMaxes
 "ArrangeBy[[Column(0)]]"
 "ArrangeBy[[Column(0)]]"
 "ArrangeBy[[Column(0)]]"
+"ArrangeBy[[Column(0)]]"
```

Comment:
This is showing all arrangements with `SET CLUSTER TO mz_introspection`, so it totally makes sense that the indexes you added had this effect.